### DISCUSSION TITLE
You are using an IPv$ address ....

### DISCUSSION THREAD

**Discussion by a4ginatl** [codefori/vscode-ibmi#2229] [Q&A]
👉🏻Why am I getting this message all of a sudden?

Why is it now important? 

![image](https://github.com/user-attachments/assets/aad95e93-efa9-4c3f-a47c-c70d3ae1286b)

<hr />

⚠️ **REMOVE THIS LINE AND ANY SENSITIVE INFORMATION BELOW!** ⚠️

|Context|Version|
|-|-|
|Code for IBM i version|2.12.1|
|Visual Studio Code version|1.92.1|
|Operating System|win32_x64|

<details>
<summary>Active extensions</summary>

```
Code Coverage for IBM i (code-coverage-ibmi): 0.1.7
Code for IBM i Walkthroughs (vscode-ibmi-walkthroughs): 0.5.0
Configuration Editing (configuration-editing): 1.0.0
Db2 for IBM i (vscode-db2i): 1.2.0
Emmet (emmet): 1.0.0
Error Lens (errorlens): 3.20.0
Git (git): 1.0.0
Git Base (git-base): 1.0.0
GitHub (github): 0.0.1
IBM i Debug (ibmidebug): 2.0.1
IBM i Notebooks (vscode-ibmi-notebooks): 0.0.6
JSON Language Features (json-language-features): 1.0.0
Live Server (LiveServer): 5.7.9
Merge Conflict (merge-conflict): 1.0.0
Node Debug Auto-attach (debug-auto-launch): 1.0.0
Pylance (vscode-pylance): 2024.8.1
Python (python): 2024.12.3
Python Test Explorer for Visual Studio Code (vscode-python-test-adapter): 0.8.2
TODO Highlight (vscode-todo-highlight): 1.0.5
Test Adapter Converter (test-adapter-converter): 0.1.9
Test Explorer UI (vscode-test-explorer): 2.21.1
TypeScript and JavaScript Language Features (typescript-language-features): 1.0.0
```
</details>

<hr />

<details>
<summary>Remote system</summary>

|Setting|Value|
|-|-|
|IBM i OS|V7R4M0|
|Tech Refresh|9|
|CCSID Origin|65535|
|Runtime CCSID|65535|
|Default CCSID|37|
|SQL|Enabled
|Source dates|Enabled

### Enabled features

|/QOpenSys/pkgs/bin|/usr/bin|/QSYS.lib/ILEDITOR.lib|/QSYS.LIB|/QIBM/ProdData/IBMiDebugService/bin|
|-|-|-|-|-|
|bash|attr|GETNEWLIBL.PGM|QZDFMDB2.PGM|startDebugService.sh|
|chsh|iconv||||
|grep|setccsid||||
|ls|tar||||
|md5sum|||||
|sort|||||
|stat|||||
|tn5250|||||
</details>

<details>
<summary>Shell env</summary>

```bash
BUILDLIB=QGPL
CURLIB=QGPL
HOME=/home/dfreinkel
HOST=172.17.72.21
LIBLS=DPOBJLIB DPPDATA DPSRCLIB LIBHTTP YAJL QTEMP QGPL OPRUTLLIB DFREINKEL
LOGIN=dfreinkel
LOGNAME=dfreinkel
MAIL=/var/spool/mail/dfreinkel
OLDPWD=/home/DFREINKEL
PASE_USRGRP_LIMITED=N
PATH=/QOpenSys/pkgs/bin:/QOpenSys/usr/bin:/usr/ccs/bin:/QOpenSys/usr/bin/X11:/usr/sbin:.:/usr/bin
PWD=/home/dfreinkel
SHELL=/QOpenSys/pkgs/bin/bash
SHLVL=1
SSH_CLIENT=10.27.230.98 64209 22
SSH_CONNECTION=10.27.230.98 64209 172.17.72.21 22
TZ=<EST>5<EDT>,M3.2.0,M11.1.0
USER=dfreinkel
USERNAME=dfreinkel
WORKDIR=/home/dfreinkel
_=/QOpenSys/pkgs/bin/env
```
</details>

<details>
<summary>Variants</summary>

```json
{
  "american": "#@$",
  "local": "#@$"
}
```
</details>

<details>
<summary>Errors</summary>

```json
[
  {
    "command": "/QOpenSys/usr/bin/qsh",
    "code": 1,
    "stderr": "CPF2111:  Library ILEDITOR already exists.\nCPC2206:  Ownership of object QZSHSYSTEM in QTEMP type *USRSPC changed.",
    "cwd": "/home/dfreinkel"
  },
  {
    "command": "/QOpenSys/usr/bin/qsh",
    "code": 1,
    "stderr": "CPF2125:  No objects deleted.\nCPC2206:  Ownership of object QZSHSYSTEM in QTEMP type *USRSPC changed.",
    "cwd": "/home/dfreinkel"
  }
]
```
</details>

**Comment by worksofliam** [comment]
> 👉🏻Why am I getting this message all of a sudden?
>
> Why is it now important?

This is related to debugging. We've found that you will get less issues with certificates when you use a hostname instead of an IP address.
